Controller/home.py

```diff
@@ -131,7 +131,6 @@ def change_status_application():
         error = "This job application could not be stored in the database. Please try again."
         return render_template('home.html', jobAddError=error)
     data = {}
-    status_change_email(application_id,session['email'],status)
```

Comment:
Is this deletion by mistake or intentional, as it doesn't seem to be related.

Reply:
@krishnasaurabh  These seem to be done by mistake. I will roll back the changes and test the functionalities again